### PR TITLE
Fix typos in Customizations How-To

### PR DESCRIPTION
**Pull request by simonflood**
**Modify this link to include the branch name, and possibly the page this PR modifies**:

https://osc.github.io/ood-documentation/latest/customizations.html

**Add your description here**

Fixes typos in the Customizations How-To page and a couple of included pages

### CODE REVIEW

File: source/customizations.rst [hunk 1]
```diff
@@ -1279,7 +1279,7 @@ However, some sites may want to disable this feature because they do not allow s
 nodes.
 
 To disable this, simply set the environment variable in the dashboards' env file
-``/etc/ood/config/apps/dashboard/env`` to a falsy value (0, false, off).
+``/etc/ood/config/apps/dashboard/env`` to a false value (0, false, off).
```
Comment (johrstrom) [inline source/customizations.rst]: This isn't a typo. `falsy` in a computer language is stuff that would evaluate to `false` like `off` or `0`.

**Comment by johrstrom** [comment]
Thanks for the many fixes! Wow, there are quite a few. Thanks again!

File: source/customizations.rst (hunk 1 above)
Comment (simonflood) [reply source/customizations.rst]: Today I learned something new!

**Comment by simonflood** [comment]
I found some more typos. Meant to submit as a new PR but ended up adding here!

**Comment by johrstrom** [comment]
Thanks, happy to have them here as well. I think I just have the 1 comment on `falsy`.

**Comment by johrstrom** [comment]
Thanks for the contribution! Sorry I didn't get around to it quicker.

**Pull request merged**